gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -403,7 +405,7 @@ public WritableByteChannel create(final StorageResourceId resourceId, CreateObje
           @Override
           public Storage.Objects.Insert createRequest(InputStreamContent inputStream)
               throws IOException {
-            return configureRequest(super.createRequest(inputStream), resourceId.getBucketName());
+            return constructRequest(super.createRequest(inputStream), resourceId.getBucketName());
```

Comment:
Not sure that `construct` is better then `configure` - because you already pass request into it, maybe leave it as is. or rename use `initializeRequest`.